liquibase-core/src/main/resources/www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.13.xsd

```diff
@@ -981,6 +981,7 @@
             <xsd:attributeGroup ref="changeAttributes"/>
             <xsd:attributeGroup ref="tableNameAttribute"/>
             <xsd:attribute name="tablespace" type="xsd:string"/>
+            <xsd:attribute name="tableType" type="xsd:string"/>
```

Comment:
The XSD change just goes into -latest, not in already-published versions like 4.13

Reply:
@MalloD12  can you remove this one as you already added it to latest?

Reply:
Removed!